presto-hive/pom.xml

```diff
@@ -290,4 +306,19 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <!-- Exclude tests against Glue from build -->
+                        <exclude>**/TestHiveClientGlueMetastore.java</exclude>
```

Comment:
I've first tried ```<excludedGroups>hive-glue</excludedGroups>``` with the ```@Test(groups = "hive-glue")``` annotation in this test but was not able to get the build to exclude it. ```AbstractTestHiveClient``` seems to be including tests from this subclass regardless since it has a different test group.